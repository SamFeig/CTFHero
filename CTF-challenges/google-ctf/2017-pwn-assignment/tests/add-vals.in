a=1
b=3
c=a+b
c
d="abc"
e=b+d
e
f="foo"
g=f+d
g
a.b.c=8
d=f+a.b.c
d
