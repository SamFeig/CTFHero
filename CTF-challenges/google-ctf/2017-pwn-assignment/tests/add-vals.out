> > > > 4
> > > "3abc"
> > > "fooabc"
> > > "foo8"
> Failed reading
